API.Tests/Services/ArchiveServiceTests.cs

```diff
@@ -291,7 +291,25 @@ public void CanParseComicInfo()
                 Web = "https://www.comixology.com/BTOOOM/digital-comic/450184"
             };
 
+            // XXX: This is not doing anything useful
             Assert.NotStrictEqual(expected, actual);
+            // XXX: This also works
+            Assert.NotStrictEqual(null, actual);
+        }
+
+        [Fact]
+        public void CanParseComicInfo2()
```

Comment:
Please make this more expressive, to help understand what this is adding versus the existing test. Like CanParseComicInfo_DefaultNumberIsBlank